.editorconfig

```diff
@@ -0,0 +1,14 @@
+# EditorConfig is awesome: http://EditorConfig.org
+
+root = true
+
+[*]
+end_of_line = lf
+insert_final_newline = false
```

Comment:
Why? Final Newlines are immensely useful and also stop Github from showing that annoying red circle

Reply:
This editorconfig was extracted from another PR where I was trying to introduce eslinting that didnt do any auto-formatting. But I agree now that its seperated the rules should lead editor towards compliance.